src/test/java/org/zalando/nakadi/domain/SubscriptionLocksCleaner.java

```diff
@@ -0,0 +1,36 @@
+package org.zalando.nakadi.domain;
+
+import org.apache.zookeeper.ZKUtil;
+import org.apache.zookeeper.ZooKeeper;
+
+import java.util.List;
+
+public class SubscriptionLocksCleaner {
+
+    public static void main(String args[]) throws Exception {
+        ZooKeeper zk = new ZooKeeper("localhost:2181", 30000, event -> {
+        });
+
+        final String contextPath = "/staging";
+
+        final List<String> locks = zk.getChildren(contextPath + "/nakadi/locks", false);
+        final List<String> subscriptions = zk.getChildren(contextPath + "/nakadi/subscriptions", false);
```

Comment:
10000 subscriptions, 100000 locks. It makes sense to use HashSet for subscriptions